<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>583</width>
    <height>553</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pbRemoveRow">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAddRow">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>10</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRemoveColumn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAddColumn">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>200</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="twData">
    <property name="geometry">
     <rect>
      <x>6</x>
      <y>6</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="twData2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>0</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="twResult">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>260</y>
      <width>191</width>
      <height>191</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAddRow2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>170</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRemoveRow2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>140</y>
      <width>31</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbAddColumn2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>200</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbRemoveColumn2">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>200</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPlus">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>50</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbMin">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pbPr">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>80</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Ã—</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>583</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
